src/Microsoft.AspNetCore.SignalR.Core/IHubClientsExtensions.cs

```diff
@@ -5,102 +5,113 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Extensions methods for <see cref="IHubClients{T}"/>.
+    /// </summary>
     public static class IHubClientsExtensions
     {
         /// <summary>
+        /// Gets a caller to all hub clients excluding the specified client connections.
         /// </summary>
-        /// <param name="hubClients"></param>
+        /// <param name="hubClients">The clients caller abstraction for a hub.</param>
         /// <param name="excludedConnectionId1">The first connection to exclude.</param>
-        /// <returns></returns>
+        /// <returns>A client caller.</returns>
         public static T AllExcept<T>(this IHubClients<T> hubClients, string excludedConnectionId1)
         {
             return hubClients.AllExcept(new [] { excludedConnectionId1 });
         }
 
         /// <summary>
+        /// Gets a caller to all hub clients excluding the specified client connections.
```

Comment:
Manual `<inheritdoc />` (aka copy-paste) for the doc changes I mentioned on the interface.